curations/npm/npmjs/@ag-grid-community/client-side-row-model.yaml

```diff
@@ -0,0 +1,9 @@
+coordinates:
+  name: styles
```

Comment:
The package name needs updating. It should be `client-side-row-model` instead `styles`.

Reply:
Since everything else is correct, I'll fix this after merging. I don't want to hold the PR up any further.